qiime/sdk/artifact.py

```diff
@@ -34,45 +35,44 @@ def _from_view(cls, view, type_, provenance):
         ----------
         view : Python object
             View to serialize.
-        type_ : qiime.plugin.Type or str
```

Comment:
We aren't using a `str` `type_` anymore, and this is a private method, so I removed for simplicity.